athena-federation-sdk/src/main/java/com/amazonaws/athena/connector/lambda/metadata/glue/DefaultGlueType.java

```diff
@@ -42,6 +42,7 @@
     BIT("boolean", Types.MinorType.BIT.getType()),
     VARBINARY("binary", Types.MinorType.VARBINARY.getType()),
     TIMESTAMP("timestamp", Types.MinorType.DATEMILLI.getType()),
+    TIMESTAMPTZ("timestamptz", Types.MinorType.TIMESTAMPMILLI.getType()),
```

Comment:
may not be possible but just a (possible dumb) thought: what if we treat timestamp as timestamptz? In other words, replace DATEMILLI with TIMESTAMPMILLI everywhere. What are the implications? If possible, does it simplify or complicate?

Reply:
This mapping seems wrong to me, we are redefining what TIMESTAMPMILLI means, why not map it to TIMESTAMPMILLI With TZ?